docs/dev/api.rst

```diff
@@ -1,9 +1,13 @@
-ListenBrainz API
-================
+ListenBrainz API for Beta
+=========================
 
 The ListenBrainz server supports the following end-points for submitting and fetching listens. All endpoints have this root URL:
 
-**Root URL**: ``https://api.listenbrainz.org``
+**Beta Root URL**: ``https://beta-api.listenbrainz.org``
+
+While we are in beta testing, to submit to our beta server use this root URL:
```

Comment:
This is a bit confusing, if we are in beta, do we want data sent to `api.listenbrainz.org` or `beta-api.listenbrainz.org`?